sdk/src/fee.rs

```diff
@@ -115,12 +145,13 @@ impl FeeStructure {
                     .unwrap_or_default()
             });
 
-        (budget_limits
-            .prioritization_fee
-            .saturating_add(signature_fee)
-            .saturating_add(write_lock_fee)
-            .saturating_add(compute_fee) as f64)
-            .round() as u64
+        FeeDetails {
+            transaction_fee: (signature_fee
+                .saturating_add(write_lock_fee)
+                .saturating_add(compute_fee) as f64)
+                .round() as u64,
```

Comment:
unrelated, but why do we round here even after removing congestion? these are all integers, right?

Reply:
Good catch! I'll patch it separately